<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>600</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_name_line_edit_1p">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="user_name_label_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>310</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>님 반갑습니다.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chat_open_btn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>들어가기</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>341</width>
        <height>291</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>버디버디.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>610</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="user_name_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>사용자 이름</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="signal_textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QScrollBar" name="signal_ScrollBar">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>450</y>
        <width>20</width>
        <height>71</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="receive_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>수신 메시지:</string>
      </property>
     </widget>
     <widget class="QLabel" name="signal_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>송신 메시지:</string>
      </property>
     </widget>
     <widget class="QScrollBar" name="receive_ScrollBar">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>16</width>
        <height>271</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="send_btn">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>450</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>전송</string>
      </property>
     </widget>
     <widget class="QListWidget" name="receive_listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>401</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="user_name_label_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>채팅방 목록</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="chat_list_tableWidget">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>120</y>
        <width>261</width>
        <height>491</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>번호</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>채팅방</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="chat_add_btn">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>채팅방 추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chat_search_btn">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>채팅방 조회</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="chat_name_line_edit">
      <property name="geometry">
       <rect>
        <x>452</x>
        <y>90</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="user_name_label_2p">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
